sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -1318,6 +1354,22 @@ def plot(
             Dict with keywords passed to the `matplotlib.pyplot.contourf`
             call for two-way partial dependence plots.
 
+        pdp_lim : dict of a tuple of 2 floats, default=None
```

Comment:
if find the type description confusing. maybe just dict ?
The expected dict keys and vals are explained in the parameter description